src/instance/InstanceImageOptions.ts

```diff
@@ -0,0 +1,40 @@
+import {ApiProperty} from '@nestjs/swagger';
+import {IsUUID, IsBoolean, IsNumber, IsOptional, IsEnum, IsString} from 'class-validator';
+import {LegalHoldStatus} from '@wireapp/core/dist/conversation/content/';
+
+export class InstanceImageOptions {
+  @ApiProperty()
+  @IsUUID('4')
+  conversationId!: string;
+
+  @ApiProperty()
+  @IsString()
+  data!: string;
+
+  @ApiProperty()
+  @IsNumber()
+  height!: number;
+
+  @ApiProperty()
+  @IsNumber()
+  width!: number;
+
+  @ApiProperty()
+  @IsString()
+  type!: string;
+
+  @ApiProperty({
+    enum: [LegalHoldStatus.UNKNOWN, LegalHoldStatus.DISABLED, LegalHoldStatus.ENABLED],
+  })
+  @IsEnum(LegalHoldStatus)
+  legalHoldStatus!: LegalHoldStatus;
```

Comment:
Shouldn't this be `LegalHoldStatus.UNKNOWN | LegalHoldStatus.DISABLED | LegalHoldStatus.ENABLED`?

Reply:
There are only these 3 states so it's fine to go with `LegalHoldStatus;`.